### PR TITLE
UI to view and release channels

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: go/channel/view_definition.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    pass
+
+
+class ReleaseChannelView(ChannelApiView):
+    view_name = 'release'
+    path_suffix = 'release/'
+
+    def post(self, request, channel):
+        channel.release(request.user_api)
+        # TODO: Clean up routing.
```
Comment (hodgestar) [inline go/channel/view_definition.py]: Shouldn't `user_api.release_tag()` do that?

Comment (jerith) [inline go/channel/view_definition.py]: Good point. I'll move the TODO.

Comment (hodgestar) [inline go/channel/view_definition.py]: By which I mean, we should make `release_tag` do that.

Comment (jerith) [inline go/channel/view_definition.py]: Actually, I'll remove the TODO and open a ticket to do this in develop.

**Comment by hodgestar** [comment]
:+1: -- TODO for releasing tags is happening in #436.

**Pull request merged**